roles/confluent.kafka_broker/templates/kafka_server_log4j.properties.j2

```diff
@@ -70,6 +70,12 @@ log4j.appender.authorizerAppender.Append=true
 log4j.appender.authorizerAppender.MaxBackupIndex={{kafka_broker.authorizer_appender_max_log_files}}
 log4j.appender.authorizerAppender.MaxFileSize={{kafka_broker.authorizer_appender_log_file_size}}
 
+log4j.appender.auditLogAppender=org.apache.log4j.DailyRollingFileAppender
```

Comment:
can you use the `RollingFileAppender` as done w the other appenders?

Also make this PR into 6.0.x?